docs/mms_on_fargate.md

```diff
@@ -0,0 +1,341 @@
+
+## Test The Inference
+
+First find the DNS name of your LB. It should be in `AWS Console -> Service -> EC2 -> Load Balancers` and click on the LB that you created.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/lb_dns.png)
+
+Now you can run the health checks using this load-balancer public DNS name, to verify that your newly created service is working:
+
+```bash
+curl InfraLb-1624382880.us-east-1.elb.amazonaws.com/ping 
+```
+
+```text
+http://infralb-1624382880.us-east-1.elb.amazonaws.com/ping
+{
+    "health": "healthy!"
+}
+```
+
+And now you are finally ready to run our inference! Let's download an example image:
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+```
+
+The image:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/kitten.jpg)
+
+The output of this query would be as follows,
+
+```bash
+curl -X POST InfraLb-1624382880.us-east-1.elb.amazonaws.com/squeezenet/predict -F "data=@kitten.jpg"
+```
+
+```text
+{
+      "prediction": [
+    [
+      {
+        "class": "n02124075 Egyptian cat",
+        "probability": 0.8515275120735168
+      },
+      {
+        "class": "n02123045 tabby, tabby cat",
+        "probability": 0.09674164652824402
+      },
+      {
+        "class": "n02123159 tiger cat",
+        "probability": 0.03909163549542427
+      },
+      {
+        "class": "n02128385 leopard, Panthera pardus",
+        "probability": 0.006105933338403702
+      },
+      {
+        "class": "n02127052 lynx, catamount",
+        "probability": 0.003104303264990449
+      }
+    ]
+  ]
+}
+```
+
+## Customize the containers to server your custom deep learning models
+
+For this section, we will show you how you could wrap the official container images with your custom configuration. 
+The custom MMS configuration will contain your custom deep-learning models, modified serving workers to suite your needs and any other
+customizations that you would require to run your "containerized inference service".
+
+To launch a service in AWS Fargate, you will need to have a container image of MMS. Lets look at how you could build a custom container.
+
+1. Create a directory to for your custom-container image work and work from that directory.
+```bash
+mkdir /tmp/custom_container
+```
+```bash
+cd /tmp/custom_continer
+```
+
+2. Download the [MMS configuration](https://s3.amazonaws.com/mms-config/mms_app_cpu.conf) for CPU.
+```bash
+wget https://s3.amazonaws.com/mms-config/mms_app_cpu.conf
+```
+Example CPU and GPU configurations are available on S3 at [CPU](https://s3.amazonaws.com/mms-config/mms_app_cpu.conf) and [GPU](https://s3.amazonaws.com/mms-config/mms_app_gpu.conf).
+Modify the downloaded configuration file and update it with the required models. For more on how to modify this check [Advanced Settings](../docker/advanced_settings.md)
```

Comment:
Should you do a for example, like putting in a new model.

Reply:
Done